.github/workflows/tag-release.yaml

```
@@ -0,0 +1,93 @@
+---
+name: Build and release versioned container images
+
+on:
+  create
+
+jobs:
+  tag_bundle:
+    name: Build and release the bundle container-image
+    if: github.repository == 'csi-addons/kubernetes-csi-addons' && github.ref_type  == 'tag'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Install Go 1.17
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+
```

```suggestion

      - name: Login to quay.io
        uses: docker/login-action@v1
        with:
          registry: quay.io
          username: ${{ secrets.QUAY_USERNAME }}
          password: ${{ secrets.QUAY_PASSWORD }}
          
```
We need to login to quay here too?
seems like this step for missed @nixpanic 
https://github.com/csi-addons/kubernetes-csi-addons/blob/main/.github/workflows/build-push.yaml#L26-L31

---

right, will update it!